src/implementations/DopplerLegacyClientImpl.ts

```diff
@@ -214,13 +238,16 @@ function parseDopplerEditorSettings(data: unknown): DopplerEditorSettings {
   // https://github.com/MakingSense/Doppler/pull/10148
   const d = objectOrEmptyObject(data);
   const promotionCodeEnabled = !!d.promotionCodeEnabled;
+
+  // TODO: analyze to use a store parse
```

Comment:
Should it still be analyzed? Or do we remove this `TODO`?

Reply:
deberiamos analizar si agregamos el flag de promocode en cada store, y directamente armamos un parse mas simple, y no dejar en duro el de mercado shop